### PR TITLE
Adding more labels

### PR DESCRIPTION
**Pull request by jolesh**
Adding more labels and label images focus on Denmark/environment

### CODE REVIEW

**Comment by jolesh** [comment]
@stephanegigandet I would love to get some feedback from you, I am unsure if I added the images in the right folder or if they belong in "en"

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -16508,6 +16508,19 @@ xx:NZ Vegetarian Society, NZVS
 en:NZ Vegetarian Society Vegan Certification, NZVS Vegan Certification
 xx:NZ Vegetarian Society Vegan Certification, NZVS Vegan Certification
 
+da:dansk vegetarisk forening
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:16511]: shouldn't it be under <en:vegetarian too?

**Review by stephanegigandet — APPROVED** [review]

File: taxonomies/labels.txt (hunk 1 above)
Comment (jolesh) [reply taxonomies/labels.txt:16511]: Yeah, I guess I could add that

**Review by VaiTon — APPROVED** [review]

**Comment by teolemon** [comment]
merged, thanks @jolesh @stephanegigandet

**Pull request merged**